app/model.py

```diff
@@ -0,0 +1,10 @@
+from app.managers import DisplayManager, PrintManager, SerializeManager
+
+
+class Book(DisplayManager, PrintManager, SerializeManager):
+    def __init__(self, title: str, content: str) -> None:
+        DisplayManager.__init__(self, content)
+        PrintManager.__init__(self, title, content)
+        SerializeManager.__init__(self, self)
+        self.title = title
+        self.content = content
```

Comment:
SRP, IRP, DIP  violation, book is handling multiple responsibilities.
and init should only handle class initialization, not handle business logic...